base/compiler/typeinfer.jl

```diff
@@ -513,6 +513,9 @@ function typeinf_edge(interp::AbstractInterpreter, method::Method, @nospecialize
             return code.rettype, mi
         end
     end
+    if ccall(:jl_get_module_infer, Cint, (Any,), method.module) == 0
+        return Any, nothing
+    end
```

Comment:
this needs to go at the top of the method (before looking at the cache), to ensure this code is stable